lib/ex_aws/rekognition.ex

```diff
@@ -2,70 +2,37 @@ defmodule ExAws.Rekognition do
   @moduledoc """
   Operations on ExAws Rekognition
   """
+  use ExAws.Utils,
+    format_type: :json,
+    non_standard_keys: %{}
```

Comment:
These 2 are pretty much redundant, but had to be defined for now due to a bug in `ExAws.Utils.__using__()`.